docs/products/kafka/kafka-connect/howto/bring-your-own-kafka-connect-cluster.rst

```diff
@@ -0,0 +1,207 @@
+
+1. Create the JDBC sink connector JSON configuration file named ``jdbc-sink-pg.json`` with the following content, replacing the placeholders ``PG_HOST``, ``PG_PORT``, ``PG_USERNAME``, ``PG_PASSWORD``, ``PG_DATABASE_NAME``, ``APACHE_KAFKA_HOST``, ``SCHEMA_REGISTRY_PORT``, ``SCHEMA_REGISTRY_USER``, ``SCHEMA_REGISTRY_PASSWORD``.
+
+.. code-block:: json
+
+    {
+        "name": "jdbc-sink-pg",
+        "config": {
+            "connector.class": "io.aiven.connect.jdbc.JdbcSinkConnector",
+            "connection.url": "jdbc:postgresql://PG_HOST:PG_PORT/PG_DATABASE_NAME?user=PG_USERNAME&password=PG_PASSWORD&ssl=required",
+            "tasks.max": "1",
+            "topics": "jdbc_sink",
+            "auto.create": "true",
+            "value.converter": "io.confluent.connect.avro.AvroConverter",
+            "value.converter.schema.registry.url": "https://APACHE_KAFKA_HOST:SCHEMA_REGISTRY_PORT",
+            "value.converter.basic.auth.credentials.source": "USER_INFO",
+            "value.converter.basic.auth.user.info": "SCHEMA_REGISTRY_USER:SCHEMA_REGISTRY_PASSWORD"
+        }
+    }
+
+2. Create the JDBC sink connector instance using Kafka Connect REST APIs
+
+.. code-block:: shell
+
+   curl -s -H "Content-Type: application/json" -X POST \
+      -d @jdbc-sink-pg.json                            \
+      http://localhost:8083/connectors/
+
+3. Check the status of the JDBC sink connector instance, ``jq`` is used to beautify the output
+
+.. code-block:: shell
+
+   curl localhost:8083/connectors/jdbc-sink-pg/status | jq
+
+The result should be similar to the following
+
+.. code-block:: json
+
+   {
+      "name": "jdbc-sink-pg",
+      "connector": {
+         "state": "RUNNING",
+         "worker_id": "10.128.0.12:8083"
+      },
+      "tasks": [
+         {
+            "id": 0,
+            "state": "RUNNING",
+            "worker_id": "10.128.0.12:8083"
+         }
+      ],
+      "type": "sink"
+   }
+
+
+Verify the JDBC connector using Karapace REST APIs
+''''''''''''''''''''''''''''''''''''''''''''''''''
+
+To verify that the connector is working, you can write messages to the ``jdbc_sink`` topic in Avro format using `Karapace REST APIs <https://github.com/aiven/karapace>`_ by following the steps below:
+
+1. Create a new **Avro schema** using the ``/subjects/`` endpoint, after changing the placeholders for ``REST_API_USER``, ``REST_API_PASSWORD``, ``APACHE_KAFKA_HOST``, ``REST_API_PORT``
+
+.. code-block:: shell
+
+   curl -X POST -H "Content-Type: application/vnd.schemaregistry.v1+json" \
+      --data '''
+         {"schema":
+            "{\"type\": \"record\",\"name\": \"jdbcsinkexample\",\"namespace\": \"example\",\"doc\": \"example\",\"fields\": [{ \"type\": \"string\", \"name\": \"name\", \"doc\": \"person name\", \"namespace\": \"example\", \"default\": \"mario\"},{ \"type\": \"int\", \"name\": \"age\", \"doc\": \"persons age\", \"namespace\": \"example\", \"default\": 5}]}"
```

Comment:
In the final text, this loses a lot of information off the right hand side. I'm assuming the reader would like to be able to read it all (I do note that *copying* it gives all the text correctly).

Reply:
The main problem is that you can't easily split the strings into multiple lines (or at least I couldn't)... do you have a suggestion for it?

Reply:
Hmm. I'll spend (no more than) 5 minutes on it after lunch - if it's not very easy I agree we shouldn't worry.

Reply:
Hmm. Given the text can be copied, and given there's a scrollbar, and given the only simple thing I can think of is to put the text into a JSON file and use `--data @jsonfile`, which would mean changing the text and then testing again, I think we just leave it as is.
The intent of the command is clear, and explained clearly, and the command as shown doesn't take too many lines, so I think this is OK.